### ISSUE TITLE
Backends for salva2d and nphysics2d

### ISSUE THREAD

**Issue by 0xh007** [AndreaCatania/amethyst_phythyst#13]
I'm in need of using the rustsim salva2d and nphysics2d libraries. It seems like the amethyst_physics nphysics back end only supports nphysics3d. I'm thinking about adding a back end for salva2d and nphysics2d. Should I expect any significant challenges with this, or should it be straight forward to follow the pattern set by the amethyst_physics nphysics back end?

**Comment by AndreaCatania** [comment]
Yes absolutely; it should be pretty easy add the support to those libs. Consider that nphysics3d and nphysics2d are almost the same.
